tests/sqllogictests/suites/base/03_common/03_0026_insert_into_tuple

```diff
@@ -26,7 +26,7 @@ select t.1, t.2, t.3, t.4, t.5, t.6 from t1
 0 200 -25.73 xyz 2022-06-01 2022-06-01 12:00:00.000000
 
 statement ok
-CREATE TABLE IF NOT EXISTS t2(id Int, t Tuple(a Bool, b Int64, c Float64, d String, e Date, f Timestamp)) Engine = Fuse
+CREATE TABLE IF NOT EXISTS t2(id Int, t Tuple(a Bool, b Int64, c Float64, d String, Date, Timestamp)) Engine = Fuse
```

Comment:
Do we need to support this mix of named and nameless `Tuple` types? I think when creating a table, if the number of `fields_name` and `fields_type` is different, we can just return an error.